app/src/main/java/co/co/nimblehq/showcases/poly/place/ui/screens/restaurant/nearby/NearbyRestaurantsScreen.kt

```diff
@@ -0,0 +1,222 @@
+            id = UUID.randomUUID().toString(),
+            thumbnailImage = ContextCompat.getDrawable(
+                LocalContext.current,
+                R.drawable.im_restaurant_mock
+            )!!.toBitmap(width = 600, height = 600),
+            name = "Dragon X",
+            address = "111, Dr. Strange street",
+            distance = "50 m"
+        ),
+        Restaurant(
+            id = UUID.randomUUID().toString(),
+            thumbnailImage = ContextCompat.getDrawable(
+                LocalContext.current,
+                R.drawable.im_restaurant_mock
+            )!!.toBitmap(width = 600, height = 600),
+            name = "Dragon X",
+            address = "111, Dr. Strange street",
+            distance = "50 m"
+        )
+    )
+
+    Scaffold(
+        bottomBar = {
+            BottomAppBar(
+                modifier = Modifier.fillMaxWidth(),
+                contentPadding = PaddingValues(bottom = dimensions.spacingSmall),
+                backgroundColor = ViolentViolet89,
+                elevation = 0.dp
+            ) {
+                Text(
+                    modifier = Modifier.fillMaxWidth(),
+                    color = Color.White,
+                    fontSize = dimensions.textSizeNormal,
+                    text = "${stringResource(id = R.string.app_version)} ${BuildConfig.VERSION_NAME}",
+                    textAlign = TextAlign.Center
+                )
+            }
+        }
+    ) {
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(it)
+                .paint(
+                    painter = painterResource(id = R.drawable.bg_nearby_restaurants),
+                    contentScale = ContentScale.FillBounds
+                )
+        ) {
+            NearbyRestaurantsToolbar()
+            NearbyRestaurantList(mockRestaurants)
+        }
+    }
+    Timber.d("Result : $uiModels")
+}
+
+@Composable
+private fun NearbyRestaurantsToolbar() {
+    ConstraintLayout(modifier = Modifier.fillMaxWidth()) {
+        val (title, iconEnd, divider) = createRefs()
+
+        val spacingNormal = dimensions.spacingNormal
+        val spacingLarge = dimensions.spacingLarge
+        val spacingHuge = dimensions.spacingHuge
+
+        Text(
+            modifier = Modifier
+                .constrainAs(title) {
+                    top.linkTo(parent.top, spacingHuge)
+                    start.linkTo(parent.start, spacingNormal)
+                },
+            text = stringResource(R.string.nearby_restaurants_title),
+            color = Color.White,
+            fontSize = dimensions.textSizeToolbar
+        )
+        Icon(
+            modifier = Modifier
+                .size(dimensions.iconSizeToolbar)
+                .constrainAs(iconEnd) {
+                    end.linkTo(parent.end, spacingLarge)
+                    centerVerticallyTo(title)
+                },
+            painter = painterResource(id = R.drawable.ic_locationing),
```

Comment:
Let's rename this drawable back to `.ic_location` 😂 
```suggestion
            painter = painterResource(id = R.drawable.ic_location),
```

Reply:
I used the same name from the exported image 😄 let me change it